### DISCUSSION TITLE
reference/docs/emoji

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#720] [Q&A]
# reference/docs/emoji

https://blog.cutterman.cn/reference/docs/emoji.html

**Comment by ang00** [comment]
11

**Comment by Cassandra44131** [comment]
It’s actually really useful to have all emoji neatly organized in one place like this — especially when you’re working on content or UI and need to quickly find the right symbol without guessing. I didn’t realize how many variations and categories there are until seeing them all together. Lately I’ve also been trying out different visual tools for profiles and content, including professional headshots (https://bestphotos.ai/headshot-generator), and it’s interesting how even small visual details can change how something feels overall. Curious though — do you think most people stick to a small set of emojis, or do they actually explore and use a wider range over time?
